vllm/_custom_ops.py

```diff
@@ -467,6 +442,32 @@ def machete_prepack_B_fake(b_q_weight: torch.Tensor,
         return torch.empty_like(b_q_weight,
                                 memory_format=torch.contiguous_format)
 
+if hasattr(torch.ops._C, "ggml_dequantize"):
```

Comment:
Leave a comment on why this can be conditional

Reply:
Originally the meta kernel was added under another conditional check, albeit checking for another op. I suppose it's so that in platforms where these kernels are not built it wouldn't run into problems loading this script. I'm not very familiar with torch's custom kernel registration, and would appreciate if you could give pointers on the proper way of handling this.

Reply:
I'm honestly not sure why but I do know that this pattern has been used before - it is a fair check to make